engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/HistoryCleanupConfigurationDto.ftl

```diff
@@ -0,0 +1,32 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/history-cleanup/get-cleanup-configuration/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto >
+    
+    <@lib.property
+        name = "batchWindowStartTime"
+        type = "string"
+        format = "date-time"
+        desc = "Start time of the current or next batch window."
+    />
+    
+    <@lib.property
+        name = "batchWindowEndTime"
+        type = "string"
+        format = "date-time"
+        desc = "End time of the current or next batch window."
```

Comment:
Should we add a hint to the default date format documentation, like we do with most other endpoints? 😃 
```suggestion
        desc = "End time of the current or next batch window. 
                     By [default](${docsUrl}/reference/rest/overview/date-format/), the date must have the format `yyyy-MM-dd'T'HH:mm:ss.SSSZ`, 
                     e.g., `2013-01-23T14:42:45.000+0200`."
```